config/user-libblkid.m4

```diff
@@ -96,17 +89,13 @@ AC_DEFUN([ZFS_AC_CONFIG_USER_LIBBLKID], [
 			[
 				rm -f $ZFS_DEV
 				AC_MSG_RESULT([no])
-				AS_IF([test "x$with_blkid" != xcheck],
-					[AC_MSG_FAILURE(
-					[--with-blkid given but unavailable])])
+				AC_MSG_FAILURE([libblkid test failed (--without-libblkid to disable)])
 			])
 
 			LIBS="$saved_LIBS"
 		],
 		[
-			AS_IF([test "x$with_blkid" != xcheck],
-				[AC_MSG_FAILURE(
-				[--with-blkid given but unavailable])])
+			AC_MSG_FAILURE([libblkid test failed (--without-libblkid to disable)])
```

Comment:
We should advise users to install libblkid and their distribution's -dev package unless they are absolutely certain that they do not want it.